add-list.csv

```diff
@@ -1,2 +1,3 @@
 nomCommune,codeCommune,codePostal,libelleAcheminement,commentaire
 Suzan,09304,09240,SUZAN,commune manquante dans le FIMOCT
+Paris 16,75116,75116,PARIS 16,arrondissement avec code postal manquant dans le FIMOCT
```

Comment:
Maybe we could use the same `nomCommune` than existing entries: `Paris 16e Arrondissement`

Reply:
From the JSON generated file `codes-postaux.json`, changing `Paris 16` to `Paris 16e Arrondissement` does not change the output but could be more useful for 3rd party using directly our repo.

So, I agree about the change. Will made it after the following discussion is set.

We may also discuss `PARIS 16` content. When using it, we get in `codes-postaux.json`

```
{"codePostal":"75013","codeCommune":"75113","nomCommune":"Paris 13e Arrondissement","libelleAcheminement":"PARIS"},
{"codePostal":"75014","codeCommune":"75114","nomCommune":"Paris 14e Arrondissement","libelleAcheminement":"PARIS"},
{"codePostal":"75015","codeCommune":"75115","nomCommune":"Paris 15e Arrondissement","libelleAcheminement":"PARIS"},
{"codePostal":"75016","codeCommune":"75116","nomCommune":"Paris 16e Arrondissement","libelleAcheminement":"PARIS 16"},
{"codePostal":"75116","codeCommune":"75116","nomCommune":"Paris 16e Arrondissement","libelleAcheminement":"PARIS 16"},
{"codePostal":"75017","codeCommune":"75117","nomCommune":"Paris 17e Arrondissement","libelleAcheminement":"PARIS"},
{"codePostal":"75018","codeCommune":"75118","nomCommune":"Paris 18e Arrondissement","libelleAcheminement":"PARIS"},```

whereas all the other `"libelleAcheminement"` are set to `"PARIS"` except the existing (before patching with `add-list.csv`, in original content from FIMOCT file) `{"codePostal":"75016","codeCommune":"75116","nomCommune":"Paris 16e Arrondissement","libelleAcheminement":"PARIS 16"},`

```
{"codePostal":"75013","codeCommune":"75113","nomCommune":"Paris 13e Arrondissement","libelleAcheminement":"PARIS"},
{"codePostal":"75014","codeCommune":"75114","nomCommune":"Paris 14e Arrondissement","libelleAcheminement":"PARIS"},
{"codePostal":"75015","codeCommune":"75115","nomCommune":"Paris 15e Arrondissement","libelleAcheminement":"PARIS"},
{"codePostal":"75016","codeCommune":"75116","nomCommune":"Paris 16e Arrondissement","libelleAcheminement":"PARIS 16"},
{"codePostal":"75116","codeCommune":"75116","nomCommune":"Paris 16e Arrondissement","libelleAcheminement":"PARIS"},
{"codePostal":"75017","codeCommune":"75117","nomCommune":"Paris 17e Arrondissement","libelleAcheminement":"PARIS"},
{"codePostal":"75018","codeCommune":"75118","nomCommune":"Paris 18e Arrondissement","libelleAcheminement":"PARIS"},```

So, should we keep `PARIS 16` or adopt `PARIS`?

Reply:
We keep `PARIS 16` for now :)